cypress/component/annot-columns.cy.js

```diff
@@ -0,0 +1,56 @@
+import AnnotatePartAnnotatedColumns from "~/components/annot-columns.vue";
+
+
+const store = {
+    commit: () => {},
+    getters: {
+        getMappedColumns: () => (activeCategory) => {
+            return [
+                    "column1",
+                    "column2",
+                    "column3"
+            ];
+        },
+        getColumnDescription: () => (columnName) => {
+            return "descr_" + columnName;
+        }
+    },
+    mutations: {
+        removeColumn: () => (columnName) => {}
+    }
+};
+
+const props = {
+    activeCategory: "someCategory"
+};
+
+
+describe("columns annotation", () =>  {
+    it("displays columns mapped to the current category and their descriptions", () => {
+        cy.mount(AnnotatePartAnnotatedColumns, {
+            computed: store.getters,
+            propsData: props
+
+        });
+        const columns = ["column1", "column2", "column3"];
```

Comment:
Yea I think I tried that but calling the function in that way returns the function (a reference to the function) instead of the list of columns.

Reply:
OK, looks like the way to do this would be to call the getter and then call the function the getter returns like so: `store.getters.getMappedColumns()("myArgument")`. But I don't think that's clearer or more maintainable than just writing the columns out as you did. So I'd say keep things as they are